parm/config/gfs/config.atmanlfv3inc

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+########## config.atmanlfv3inc ##########
+# Atm Var Analysis specific
+
+echo "BEGIN: config.atmanlfv3inc"
+
+# Get task specific resources
+. "${EXPDIR}/config.resources" atmanlfv3inc
+
```

Comment:
How about define `JEDIYAML` and `JEDIEXE` in this script for the yaml and executable used by this job.